frontend-app/src/Thermometer.js

```diff
@@ -0,0 +1,40 @@
+import { useState } from "react";
+
+
+function Thermometer() {
+   const [temperatureValue, setTemperatureValue] = useState(10);
+ const [temperatureColor, setTemperatureColor] = useState('cold');
+
+ const increaseTemperature = () => {
+  const newTemperature = temperatureValue + 1;
+  setTemperatureValue(newTemperature);
+
+  if (newTemperature >= 15) {
+   setTemperatureColor('hot');
+  }
+ };
+
+ const decreaseTemperature = () => {
+  const newTemperature = temperatureValue - 1;
+  setTemperatureValue(newTemperature);
+  if (newTemperature < 15) {
+   setTemperatureColor('cold');
+  }
+ };
+
+ return (
+  <div className='app-container'>
+   <div className='temperature-display-container'>
+    <div className={`temperature-display ${temperatureColor}`}>{temperatureValue}°C</div>
```

Comment:
Думаю состояние для цвета лишнее
можно сделать через условие
```suggestion
    <div className={`temperature-display ${temperatureValue >= 15 'hot' : 'cold'}`}>{temperatureValue}°C</div>
```